automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -45,6 +45,25 @@ case class ClusterRequest(bucketPath: GcsBucketName,
                           labels: LabelMap,
                           jupyterExtensionUri: Option[GcsPath])
 
+case class DefaultLabels(clusterName: ClusterName,
+                         googleProject: GoogleProject,
+                         googleBucket: GcsBucketName,
+                         serviceAccount: GoogleServiceAccount,
+                         notebookExtension: Option[GcsPath]) {
+
+  // TODO don't hardcode fields
```

Comment:
Swatomation doesn't have Spray so we can't do the same thing here

Reply:
😢

Reply:
As I understand it, this is the Swat team's preference: we want to be explicit about our JSON, rather than assuming our Spray implicits are doing the right thing.